### PR TITLE
Use mintedTotallyByBridge for bridge-ui statistics

### LINKED ISSUE #110: mintedTotally is used in UI instead of mintedTotallyByBridge
Currently `mintedTotally` is used to get the value of coins minted by `erc-to-native` bridge. But in common case it is not correct since it will shown value of coins minted by all bridges if more than one bridge is able to transfer tokens to the chain.

https://github.com/poanetwork/tokenbridge/blob/26ff22ac7ecc02027f7bcc2057f77b27f94a5c10/ui/src/stores/HomeStore.js#L256

The BlockReward contract provides ability to get number of coins minted by a specific bridge by calling `mintedTotallyByBridge`.

### PR DESCRIPTION
**Pull request by patitonar**
Closes #110

### CODE REVIEW

File: ui/src/stores/utils/contract.js [hunk 1]
```diff
@@ -53,8 +53,8 @@ export const getBalanceOf = async (contract, address) => {
   return fromDecimals(balance, decimals)
 }
 
-export const mintedTotally = async contract => {
```
Comment (akolotov) [inline ui/src/stores/utils/contract.js]: is it possible to reflect this in tests?

Comment (patitonar) [reply ui/src/stores/utils/contract.js]: We could add some unit tests for this method, mocking the returned value from `contract.methods.mintedTotallyByBridge(bridgeAddress).call()` and also unit tests for `HomeStore` that could make sure that `mintedTotallyByBridge` method is called from the store

Comment (akolotov) [reply ui/src/stores/utils/contract.js]: yes, do this please

Comment (patitonar) [reply ui/src/stores/utils/contract.js]: Done

Comment (akolotov) [reply ui/src/stores/utils/contract.js]: thanks

**Pull request merged**